pkg/autoscaler/autoscaler.go

```diff
@@ -24,6 +24,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/wacul/ptr"
```

Comment:
🤦‍♂ Can we just add this to `knative/pkg` rather than juggling packages for this?

Reply:
I don't think we need it at all. A local variable will do as well.

Reply:
That would work too :)

Reply:
https://github.com/knative/pkg/pull/519/files 
Well,  as soon as matt does this!

Reply:
should be done